### PR TITLE
Modify S5847: Change text to LayC format (APPSEC-1214)

### PR DESCRIPTION
**Pull request by gaetan-ferry-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S5847/cfamily/rule.adoc [hunk 1]
```diff
@@ -1,69 +1,69 @@
...
-* https://cwe.mitre.org/data/definitions/367[MITRE, CWE-367] - Time-of-check Time-of-use (TOCTOU) Race Condition
-* https://wiki.sei.cmu.edu/confluence/display/c/FIO45-C.+Avoid+TOCTOU+race+conditions+while+accessing+files[CERT, FIO45-C.] - Avoid TOCTOU race conditions while accessing files
+=== Documentation
+
+* Carnegie Mellon University Software Engineering Institure - https://wiki.sei.cmu.edu/confluence/display/c/FIO45-C.+Avoid+TOCTOU+race+conditions+while+accessing+files[FIO45-C. - Avoid TOCTOU race conditions while accessing files]
+
+=== Standards
+
+* OWASP - https://owasp.org/Top10/A01_2021-Broken_Access_Control/[Top 10 2021 Category A1 - Broken Access Control]
+* OWASP - https://owasp.org/www-project-top-ten/2017/A5_2017-Broken_Access_Control[OWASP Top 10 2017 Category A5 - Broken Access Control]
```
Comment (sebastien-andrivet-sonarsource) [inline rules/S5847/cfamily/rule.adoc:65]: The OWASP inside `[` ... `]` should be removed

**Review by sebastien-andrivet-sonarsource — CHANGES_REQUESTED** [review]
Is the Java rule deleted on purpose (because it is not implemented)?

**Comment by gaetan-ferry-sonarsource** [comment]
@sebastien-andrivet-sonarsource The Java spec was indeed removed on purpose. As it is not implemented I did not find any value in migrating its description. There was also no point in keeping the old content.

**Review by sebastien-andrivet-sonarsource — APPROVED** [review]
LGTM but I think you forgot to remove the "OWASP".

**Pull request merged**